Dockerfile

```
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=16.20.2-bullseye-slim
+ARG NODE_VERSION=22.14.0-bookworm
```

should still use the `-slim` variant
```suggestion
ARG NODE_VERSION=22.14.0-bookworm-slim
```